### ISSUE TITLE
Focus after End timeline element disappears later it appears after pressing Tab key twice and also reverse traversed focus trapped when we are using highstock chart in kendowindow UI

### ISSUE THREAD

**Issue by vimalmavanicasepoint** [highcharts/highcharts#20839]
#### Expected behaviour
when we are using highstock chart in kendowindow UI
The focus must be moved to next element but skip the element, and also focus must work in reverse 

#### Actual behaviour
when we are using highstock chart in kendowindow UI
The focus moved to  class "highcharts-exit-anchor"
focus trapped when reverse traversed 

#### Live demo with steps to reproduce
template : https://jsfiddle.net/Vimal_Casepoint/xrchjvu6/2/

**Focus apperes when tab key twise**
1. open kendo window
2. start focus in highstock chart
3. when focus end of timeline press tab key focus will not appear to next element
4. again press tab ket focus will appear

**Reversed focus Trapped**
1. open kendo window
2. Press tab key and travel forward focus
3. when focus appear on after highstokc chart
4. Try to resvesed focus in chart
5. Focus will not move first element of popup

#### Product version
Latest version

**Comment by karolkolodziej** [comment]
Hi @vimalmavanicasepoint! 

Thank you for reporting the issue. I was able to recreate it- please see the console of [the simplified demo.](https://jsfiddle.net/BlackLabel/ne579rbd/show)
I'll forward this to our a11y team to asses it further.

**Comment by vimalmavanicasepoint** [comment]
Hi @karolkolodziej 
I'm reaching out to request an update on the above issue, Would you mind providing insights into its current status and the anticipated timeline for resolution?
Thank You sincerely for your assistance.

**Comment by oysteinmoseng** [comment]
Hi @vimalmavanicasepoint - we will investigate the issue and report back here, thanks for bringing it to our attention.

**Comment by marvin19** [comment]
Hi, @vimalmavanicasepoint 

I've been looking into this and the issue is dependent on if the modal in Kendowindow is set to `true `or `false`. It is working when it is set to false. The issue is a conflict between keyboard handling in Highcharts and Kendowindow modal handling. 

https://jsfiddle.net/maritavindedal/xjsuf1gq/

**Comment by vimalmavanicasepoint** [comment]
@marvin19 
Thank You for updating 
According to our requirements, it is not possible to set the model to false. I have already attempted this, but we cannot set the model to false.

**Comment by vimalmavanicasepoint** [comment]
@marvin19 @highsoft-asagh-bot 
I'm reaching out to request an update on the above issue, Would you mind providing insights into its current status and the anticipated timeline for resolution?
Thank You sincerely for your assistance.

**Comment by marvin19** [comment]
@vimalmavanicasepoint I've tried creating a workaround for this, but it seems to be very challenging. This might be a Kendowindow UI issue, since keyboard navigation is working as expected when turning the modal off.
